app-frontend/src/app/components/toolSearch/toolSearch.html

```diff
@@ -13,9 +13,10 @@
                  id="search-input"
                  class="form-control input-dark"
                  placeholder="Search for tools"
-                 ng-model="$ctrl.searchText"/>
-          <button type="button" class="btn btn-link">
-            <span class="close">x</span>
+                 ng-model="$ctrl.searchText"
+                 ng-keyup="$event.keyCode == 13 && $ctrl.onSearchAction()"/>
```

Comment:
The functionality looks good, but I think we are a bit far away from being able to support search on the backend so this might be a good candidate to put behind a feature flag for now